### ISSUE TITLE
Is there an easy way to set IssuedAt field from the current time?

### ISSUE THREAD

**Issue by chmorgan** [dgrijalva/jwt-go#342]
Would now be something like time.Now().Unix()?

**Comment by dgrijalva** [comment]
That would do it. I'd probably do `time.Now().UTC().Unix()` to avoid any timezone issues.

**Comment by chmorgan** [comment]
Doh, good call on the UTC, I was doing it without that.

**Comment by dgrijalva** [comment]
Yeah. It's not strictly necessary if the machine the app runs on is already in UTC, which is likely for a server. That may not always be the case (dev environments, configuration mishaps, unique environments) so it's worth being explicit where it matters.
